<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loyality_points</class>
 <widget class="QDialog" name="loyality_points">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>تعريف نقاط العضويه</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <widget class="QLabel" name="desc_3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>110</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>صالح من تاريخ</string>
   </property>
  </widget>
  <widget class="QLabel" name="desc_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>صالح الي تاريخ</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="Qdate_to">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>79</width>
     <height>18</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="minimumDateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2020</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="Qdate_from">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>79</width>
     <height>18</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="minimumDateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2020</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>70</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>قيمه النقاط</string>
   </property>
  </widget>
  <widget class="QLabel" name="desc_11">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>30</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>عدد النقاط</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="Qtable_point">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>491</width>
     <height>281</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <column>
    <property name="text">
     <string>المسلسل</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>كميه النقاط</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>قيمه النقاط</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>من تاريخ</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>إلى تاريخ</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="BTN_createLoyPoint">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>82</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width: 6em;
                                        }
                                        QPushButton:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                        QPushButton:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    </string>
   </property>
   <property name="text">
    <string>إنشاء</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_modifyLoyPoint">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>82</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width: 6em;
                                        }
                                        QPushButton:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                        QPushButton:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    </string>
   </property>
   <property name="text">
    <string>تعديل</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_searchLoyPoint">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>82</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width: 6em;
                                        }
                                        QPushButton:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                        QPushButton:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    </string>
   </property>
   <property name="text">
    <string>بحث</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_searchLoyPoint_all">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>82</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
                                        background-color: white;
                                        border-style: outset;
                                        border-width: 2px;
                                        border-radius: 10px;
                                        border-color: blue;
                                        min-width: 6em;
                                        }
                                        QPushButton:hover {
                                        background: qradialgradient(
                                        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
                                        );
                                        }
                                        QPushButton:pressed {
                                        border-style: inset;
                                        background: qradialgradient(
                                        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
                                        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
                                        );
                                        }
                                    </string>
   </property>
   <property name="text">
    <string>عرض الكل</string>
   </property>
  </widget>
  <widget class="QLabel" name="LB_pointId">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="LE_pointValue">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="maximum">
    <double>9.990000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="LE_pointQty">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99999</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
